app/views/sandbox/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= content_for :title, t('page_titles.sandbox') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= t('page_titles.sandbox') %></h1>
+
+    <p class="govuk-body">The API sandbox is a sandbox environment which allows you to experience the participant events.</p>
+
+    <p class="govuk-body">You can:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Authenticate</li>
+      <li>Test the API <%= govuk_link_to 'API integrations', '/api-docs' %></li>
+    </ul>
+
+    </div>
+</div>
```

Comment:
Couple of general points:
1. We try to use double quotes in views where possible.
2. Either all of it should be internationalised or none of it should imo.
3. Second-to-last `div` is too indented :)

Reply:
Ok, but this will completely change once we get right content. This is just to put a landing page for sandbox web app.

Reply:
I find the first point odd here. Single quotes parse faster, since Ruby is expecting interpolation if double quotes are used. It does appear that Rubocop has been set up that way, which still gets me every time.

Reply:
I guess the speed bump is negligible compared to everything else that a web app is doing, and we use interpolation quite a lot. And having both types of quotes looks messy.